source/rhoas/src/main/java/com/openshift/cloud/controllers/ConditionUtil.java

```diff
@@ -224,6 +224,11 @@ public static String getStandarizedErrorMessage(int code, Exception e) {
         TypeReference<HashMap<String, Object>> typeRef = new TypeReference<>() {};
         errorObject = new ObjectMapper().readValue(e.getMessage(), typeRef);
         reason = errorObject.getOrDefault("reason", "").toString();
+
+        if (reason.isBlank()) {
+          reason = errorObject.getOrDefault("error_description", "").toString();
```

Comment:
This is generic method for handling conditions but here we using field that is probably specific for some API request. 
Why here then and not in the `AccessTokenSecretTool` file. Is this like futureproofing things?

Reply:
No, this was a very quick hack. Looking more closely at it we probably shouldn't be using kas ApiException for an auth exception. I'll refactor today.